context-free.md

```diff
@@ -1,22 +1,36 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+_This programming language is for artists who want to render images with
+repeated patterns and who have some understanding of programming._
 
 ## What is easy to do in this language? Why is it easy?
-
+_It's very easy to repeat a single image or a pattern of images. This is
+because the rules in Context Free allow for the same pattern to be
+copied in one line of code. These rules also allow the user to change the
+location, color, and size of the image by only changing parameters._
 
 ## What is hard to do in this language? Why is it hard?
-
+_It's hard to make an image that has many individual pieces that are not the
```

Comment:
Why do you think this might be? Generally when you hit something hard like this, it means you're doing something that the creators of the language were trying to steer you away from (or they designed their language poorly). Why might they have made this tradeoff?